src/app/gulpfile.js

```diff
@@ -14,6 +14,13 @@ const collectionEditor = process.env.sunbird_collection_editor_artifact_url;
 const genericEditor = process.env.sunbird_generic_editor_artifact_url;
 const editorsDestPath = 'client/src/thirdparty/editors/'
 
+// PhraseApp Configuration
+const authToken = 'Basic YXJqdW5raGVjeaE6QXJqdW5AMTk5MA==';
```

Comment:
Configuration cannot be directly used here; it will be public.

CC: @rajeevsathish

Reply:
Done bro.